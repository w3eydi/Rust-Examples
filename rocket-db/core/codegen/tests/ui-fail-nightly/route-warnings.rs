../ui-fail/route-warnings.rs