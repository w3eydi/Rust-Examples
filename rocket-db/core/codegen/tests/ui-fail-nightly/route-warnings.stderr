warning: 'application/x-custom' is not a known media type
 --> $DIR/route-warnings.rs:7:21
  |
7 | #[get("/", format = "application/x-custom")]
  |                     ^^^^^^^^^^^^^^^^^^^^^^

warning: 'x-custom/plain' is not a known media type
  --> $DIR/route-warnings.rs:10:21
   |
10 | #[get("/", format = "x-custom/plain")]
   |                     ^^^^^^^^^^^^^^^^

warning: 'x-custom/x-custom' is not a known media type
  --> $DIR/route-warnings.rs:13:21
   |
13 | #[get("/", format = "x-custom/x-custom")]
   |                     ^^^^^^^^^^^^^^^^^^^

warning: `data` used with non-payload-supporting method
  --> $DIR/route-warnings.rs:18:12
   |
18 | #[get("/", data = "<_foo>")]
   |            ^^^^^^^^^^^^^^^
   |
note: 'GET' does not typically support payloads
  --> $DIR/route-warnings.rs:18:3
   |
18 | #[get("/", data = "<_foo>")]
   |   ^^^
   = note: this warning originates in the attribute macro `get` (in Nightly builds, run with -Z macro-backtrace for more info)

warning: `data` used with non-payload-supporting method
  --> $DIR/route-warnings.rs:21:13
   |
21 | #[head("/", data = "<_foo>")]
   |             ^^^^^^^^^^^^^^^
   |
note: 'HEAD' does not typically support payloads
  --> $DIR/route-warnings.rs:21:3
   |
21 | #[head("/", data = "<_foo>")]
   |   ^^^^
   = note: this warning originates in the attribute macro `head` (in Nightly builds, run with -Z macro-backtrace for more info)

error: checking for warnings!
  --> $DIR/route-warnings.rs:25:5
   |
25 |     compile_error!("checking for warnings!")
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
